libvast_test/src/events.cpp

```diff
@@ -107,7 +107,7 @@ inhale<format::json::reader>(const char* filename,
   caf::put(settings, "vast.import.json.selector", "event_type:suricata");
   auto input = std::make_unique<std::ifstream>(filename);
   format::json::reader reader{settings, std::move(input)};
-  reader.module(events::suricata_module);
+  CHECK(!reader.module(events::suricata_module));
```

Comment:
This should compare against `caf::error{}` so the error is shown if there is one. It should also be `REQUIRE_*` rather than `CHECK_*` as it doesn't make sense to continue test execution if this fails.